src/ResourceManager/Automation/Commands.Automation/Cmdlet/NewAzureAutomationSchedule.cs

```diff
@@ -136,6 +136,12 @@ public NewAzureAutomationSchedule()
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The schedule time zone.")]
         public string TimeZone { get; set; }
 
+        /// <summary>
+        /// Passed to indicate that the schedule object returned by the cmdlet is to be used for update management
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Indicates that this schedule object will be used for scheduling an update deployment")]
+        public SwitchParameter ForUpdate { get; set; }
```

Comment:
@vrdmr @safeermohammed : Is this a new parameter for this cmdlet? If it is, it need to go to a cmdlet review

Reply:
😮 I found the design review and comment specifically for this:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/58#issuecomment-386468791
Based on that design review, I'd definitely name this `ForUpdateConfiguration` so that the intent is obvious. Additionally, I don't see any help documentation changes in this PR for this parameter. You may need to regenerate the help documentation.